src/app/QuickStarts/quickstarts/automated-rules-quickstart.tsx

```diff
@@ -31,7 +31,7 @@ const AutomatedRulesQuickStart: QuickStart = {
     durationMinutes: 5,
     icon: <CryostatIcon />,
     description: `Learn about automated rules in **[APP]** and how to create one.`,
-    prerequisites: ['Start a Recording'],
+    prerequisites: ['Start a recording'],
```

Comment:
Seems inconsistent - elsewhere we treat recordings as a resource and capitalize them as Recordings, but here this is reversed?

Reply:
yes, I missed this.